src/Microsoft.Net.Http.Headers/ContentDispositionHeaderValueExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Net.Http.Headers
+{
+    public static class ContentDispositionHeaderValueExtensions
```

Comment:
Rename: Describe the functionality being added, not just the type being extended. Otherwise we get type conflicts.